Starter devel 2

ianabc opened the pull request:
Some changes to the upstream of drupal-vm have broken our current configuration. I'm adding a template config.yml and drupal.make.yml to the starter directory along with a README with some hints for getting things going on OS X.

ianabc committed:
Updated default.settings.local.php to reflect drupal-vm config a bit

Carlin helped Gerry and I get drupal-vm configured, this covers some of
the sticking points we had.

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc committed:
Ignoring modules added by vagrant

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc committed:
Merge branch 'develop' of https://github.com/ianabc/away into develop

* 'develop' of https://github.com/ianabc/away:
  Updated default.settings.local.php to reflect drupal-vm config a bit

ianabc committed:
Adding configs

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc committed:
Adding README

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc committed:
Correcting a couple of mistakes in the README

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc committed:
Adding code blocks

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc committed:
Playing with fencing

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc committed:
Still playing

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc committed:
Come on!

Signed-off-by: Ian Allison <iana@pims.math.ca>

joelpittet commented on public/sites/all/modules/contrib/.gitignore:
```diff
@@ -0,0 +1,2 @@
+backup_migrate
```
This one is good to have in all environments, probably shouldn't ignore, no?

joelpittet commented on public/sites/all/modules/contrib/.gitignore:
```diff
@@ -0,0 +1,2 @@
+backup_migrate
+devel
```
We should have a `public/sites/all/modules/dev/` folder that setup like the `working` folder and we can include development modules in there

joelpittet commented:
Other than the git ignore this is a great idea. Thanks @ianabc

ianabc committed:
Wrong place to do that

Signed-off-by: Ian Allison <iana@pims.math.ca>

ianabc commented:
No problem, I've removed the gitignore, I was just confused because drupalvm was trying to pull in backup_migrate and devel. I've removed them from the drupal.make.yml for the moment, backup_migrate was already included in 2a953abeef10180d0003881460eb037435972509, and I guess devel could go in as another pull request.
